README.md

```diff
@@ -22,4 +22,19 @@ mkctr \
 `mkctr` auto discovers `GOOS`/`GOARCH` from the specified base image. If the base image supports multiple platforms, binaries are compiled for each platform as long as it's one of `linux/amd64`, `linux/386`, `linux/arm`, `linux/arm64`.
 
 ## Maturity
-This is under active development. While Tailscale uses it, backwards compatability is not guaranteed, and some functionality is missing.
\ No newline at end of file
+
+This is under active development. While Tailscale uses it, backwards compatability is not guaranteed, and some functionality is missing.
+
+## Known issues
+
+Base images with media type `application/vnd.docker.distribution.manifest.v2+json` do not work with `mkctr`.
+
+Known supported base image media types are `application/vnd.oci.image.index.v1+json`.
+
+You can verify the media type of your image using [gcrane](https://github.com/google/go-containerregistry/tree/main/cmd/gcrane) and [jq](https://jqlang.github.io/jq/download/):
+
+```
+gcrane manifest <image> | jq .mediaType
+```
+
+If you are building the base image with `docker buildx build ...` make sure you don't pass `--load` or `--output=type=docker` flags that would result in an image of docker media type.
```

Comment:
```suggestion
If you are building the base image with `docker buildx build ...` passing `--load` or `--output=type=docker` flags is known to result in an unsupported media type.
```

(just because it took me a few seconds to understand that "docker media type" refers to `application/vnd.docker.distribution.manifest.v2+json`)

Reply:
yeah I don't think 'docker media type' is really a thing